src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -1793,7 +1793,10 @@ void EmitUpdateBumpalong(RegexNode node)
                 Debug.Assert(node.Type is RegexNode.UpdateBumpalong, $"Unexpected type: {node.Type}");
 
                 TransferSliceStaticPosToPos();
-                writer.WriteLine("base.runtextpos = pos;");
+                using (EmitBlock(writer, "if (base.runtextpos < pos)"))
```

Comment:
Source generator doesn't support Right to left anymore right? Just making sure since if not, then this condition would probably depend on that.

Reply:
Correct